src/py/flwr/common/serde_test.py

```diff
@@ -180,10 +187,132 @@ def test_named_values_serialization_deserialization() -> None:
             assert expected == actual
 
 
+T = TypeVar("T")
+
+
+class RandomMaker:
```

Comment:
Can we split the `RandomMaker`-related refactoring into a separate PR?

I like the simplification this provides, but it would be better to make our tests reproducible (i.e., remove randomness).

Reply:
Just updated the naming. The class is determinant.